docs/shared/packages/webpack/webpack-config-setup.md

```diff
@@ -166,12 +166,12 @@ const { merge } = require('webpack-merge');
 const withModuleFederation = require('@nrwl/react/module-federation');
 // or `const withModuleFederation = require('@nrwl/angular/module-federation');`
 
-module.exports = composePlugins(withNx(), (config, { options, context }) => {
+module.exports = composePlugins(withNx(), async (config, { options, context }) => {
   const federatedModules = await withModuleFederation({
     // your options here
   });
 
-  return merge(federatedModules(config, context), {
+  return merge(federatedModules(config, { options, context }), {
```

Comment:
`withModuleFederation` returns `Promise<AsyncNxWebpackPlugin>`:
https://github.com/nrwl/nx/blob/b87f838e9a5a1bfb500ba0027c8604dea9492071/packages/react/src/module-federation/with-module-federation.ts#L29-L31
`AsyncNxWebpackPlugin`'s second argument is `NxWebpackExecutionContext`:
https://github.com/nrwl/nx/blob/b87f838e9a5a1bfb500ba0027c8604dea9492071/packages/webpack/src/utils/config.ts#L26-L30
`NxWebpackExecutionContext`:
https://github.com/nrwl/nx/blob/b87f838e9a5a1bfb500ba0027c8604dea9492071/packages/webpack/src/utils/config.ts#L18-L21